### PR TITLE
Improved some user UI

### PR DESCRIPTION
**Pull request by Olegas123**
Improved some main page UI

### CODE REVIEW

File: Kinder/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -1,2 +1,58 @@
...
+        How does this application work?
+    </div>
+    <div class="par7">
+        This application works on the concept of "swiping" that is being used in some other applications like "Tinder", so thats why it is called Kinder :)
+    </div>
+    <div class="par1">
+        How do I help out / get items?
+    </div>
+    <div class="par8">
+        It is easy! Only 3 steps to become a member! Step 1: Login or create a new account! Step 2 Create an item description and publish it, or swipe other people items in order to find the one thet you need! Step 3 Get or give away items that You want!
```
Comment (JustasBan) [inline Kinder/Views/Home/Index.cshtml]: add some br here, like in design

File: Kinder/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -1,2 +1,58 @@
...
+    </div>
+    <div class="rect4">
+        <img src="~/Rectangle 4.png" />
+    </div>
+    <div class="par6">
+        So in order to fight this problem, our team created an application which will help people to give away some items that they dont need anymore,
+        and also to give them not for utilization, but for those ones who really need something.
+    </div>
+    <div class="par3">
+        Give it a try
```
Comment (JustasBan) [inline Kinder/Views/Home/Index.cshtml]: make this a link to swiping

File: Kinder/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -1,2 +1,58 @@
...
+    <div class="par8">
+        It is easy! Only 3 steps to become a member! Step 1: Login or create a new account! Step 2 Create an item description and publish it, or swipe other people items in order to find the one thet you need! Step 3 Get or give away items that You want!
+    </div>
+</body>
+</html>
+
+
+
+
+
```
Comment (JustasBan) [inline Kinder/Views/Home/Index.cshtml]: excessive lines

**Comment by JustasBan** [comment]
also, wrap everything after photos section into center tag, so it will be naturally centered

**Comment by JustasBan** [comment]
fix footer too, or delete it, it doesn't serve much purpose

File: Kinder/Views/Home/Index.cshtml [hunk 4]
```diff
@@ -1,2 +1,58 @@
...
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="~/css/mainStyle.css" />
+    <link rel="stylesheet" href="~/css/site.css" />
+</head>
+<body class="mainBackground">
+    <div>
+        <p class="welcome">
+            Welcome to Kinder! Give away. Get karma points. Compete with other people!
```
Comment (The-Blue-K) [inline Kinder/Views/Home/Index.cshtml]: Welcome to Kinder!
Give, get karma points and compete with other people!

File: Kinder/Views/Home/Index.cshtml [hunk 5]
```diff
@@ -1,2 +1,58 @@
...
+        </p>
+    </div>
+    <div class="rect">
+        <img src="~/Rectangle 3.png" />
+    </div>
+    <div class="rect1">
+        <img src="~/Rectangle 1.png" />
+    </div>
+    <div class="par4">
+        Each year, a lot of variuos trash is being produced. And sometimes, items that appear in our trash container, are not actually a real trash!
```
Comment (The-Blue-K) [inline Kinder/Views/Home/Index.cshtml:23]: Each year, a lot of various items are thrown away, and sometimes, these items that appear in our garbage cans, are not actually garbage! Some people just want to free up some space by getting rid of their various unused items, but they don't stop to think that their trash could be someone else's treasure.

Comment (The-Blue-K) [inline Kinder/Views/Home/Index.cshtml]: Each year, a lot of various items are thrown away, and sometimes, these items that appear in our garbage cans, are not actually garbage!

File: Kinder/Views/Home/Index.cshtml [hunk 6]
```diff
@@ -1,2 +1,58 @@
...
+    </div>
+    <div class="par2">
+        Some people just want to free some space from their variuos items, which are not being used by themselves
+        , but can be useful for some people.
```
Comment (The-Blue-K) [inline Kinder/Views/Home/Index.cshtml]: Some people just want to free up some space by getting rid of their various unused items, but they don't stop to think that their trash could be someone else's treasure.

File: Kinder/Views/Home/Index.cshtml [hunk 7]
```diff
@@ -1,2 +1,58 @@
...
+    </div>
+    <div class="rect4">
+        <img src="~/Rectangle 4.png" />
+    </div>
+    <div class="par6">
+        So in order to fight this problem, our team created an application which will help people to give away some items that they dont need anymore,
+        and also to give them not for utilization, but for those ones who really need something.
```
Comment (The-Blue-K) [inline Kinder/Views/Home/Index.cshtml]: So in order to fight this problem, our team created an application which will help people give away items that they don't need anymore to someone who really does.

File: Kinder/Views/Home/Index.cshtml [hunk 8]
```diff
@@ -1,2 +1,58 @@
...
+    </div>
+    <div class="par3">
+        Give it a try
+    </div>
+    <div class="par5">
+        How does this application work?
+    </div>
+    <div class="par7">
+        This application works on the concept of "swiping" that is being used in some other applications like "Tinder", so thats why it is called Kinder :)
```
Comment (The-Blue-K) [inline Kinder/Views/Home/Index.cshtml]: This application works on the concept of "swiping" that can be seen in some other applications like "Tinder" so the app should feel familiar to anyone that has seen or used such apps.

Comment (The-Blue-K) [reply Kinder/Views/Home/Index.cshtml]: List items that you don't need
People who need them will like them
Choose who will get your item
Earn karma points and compete with other people on the leaderboards!

File: Kinder/Views/Home/Index.cshtml (hunk 1 above)
Comment (The-Blue-K) [inline Kinder/Views/Home/Index.cshtml]: Step 1
Login or create a new account!

Step 2
Create an item description and publish it, or swipe other people items in order to find the one that you need!

Step 3
Get the items you want or give away the ones you don't!

**Comment by The-Blue-K** [comment]
The whole corrected description text:

Welcome to Kinder!
Give, get karma points and compete with other people!
Each year, a lot of various items are thrown away, and sometimes, these items that appear in our garbage cans, are not actually garbage! Some people just want to free up some space by getting rid of their various unused items, but they don't stop to think that their trash could be someone else's treasure.

So in order to fight this problem, our team created an application which will help people give away items that they don't need anymore to someone who really does.

This application works on the concept of "swiping" that can be seen in some other applications like "Tinder" so the app should feel familiar to anyone that has seen or used such apps.

List items that you don't need
People who need them will like them
Choose who will get your item
Earn karma points and compete with other people on the leaderboards!

Tutorial:

Step 1
Login or create a new account!

Step 2
Create an item description and publish it, or swipe other people items in order to find the one that you need!

Step 3
Get the items you want or give away the ones you don't!

File: Kinder/Views/Home/Index.cshtml (hunk 1 above)
Comment (Olegas123) [reply Kinder/Views/Home/Index.cshtml]: Added

File: Kinder/Views/Home/Index.cshtml (hunk 2 above)
Comment (Olegas123) [reply Kinder/Views/Home/Index.cshtml]: Done

File: Kinder/Views/Home/Index.cshtml (hunk 3 above)
Comment (Olegas123) [reply Kinder/Views/Home/Index.cshtml]: Done

**Comment by Olegas123** [comment]
Changed everything to according comments

**Pull request merged**